packages/esm-tb-app/src/views/program-management/maternal-health.component.tsx

```diff
@@ -1,27 +1,37 @@
 import React from 'react';
 import { Tabs, Tab, TabList, TabPanels, TabPanel } from '@carbon/react';
 import styles from '../common.scss';
-import { useTranslation } from 'react-i18next';
-import MdrTbList from './tabs/mdr-tb.component';
-import TbPatientTracing from './tabs/tb-patient-tracing.component';
-import { PatientChartProps } from '@ohri/openmrs-esm-ohri-commons-lib';
+import { PatientChartProps, getMenuItemTabConfiguration, EncounterList } from '@ohri/openmrs-esm-ohri-commons-lib';
+import tptProgramManagemetConfigSchema from './tb-program-management-config.json';
+interface OverviewListProps {
+  patientUuid: string;
+}
 
 const ProgramManagementSummary: React.FC<PatientChartProps> = ({ patientUuid }) => {
```

Comment:
Why is this component named maternal-health.component.tsx shouldn't be something like tb-program-management.component.tsx